keras/src/backend/openvino/numpy.py

```diff
@@ -1579,7 +1622,31 @@ def logical_or(x1, x2):
     return OpenVINOKerasTensor(ov_opset.logical_or(x1, x2).output(0))
 
 
-def logspace(start, stop, num=50, endpoint=True, base=10, dtype=None, axis=0):
+def logspace(
+    start,
+    stop,
+    num=50,
+    endpoint=True,
+    base=10,
+    dtype=None,
+    axis=0,
+):
+    """Return numbers spaced evenly on a log scale.
+
+    OpenVINO backend currently supports only axis=0 and axis=-1.
+    """
+
+    # Validate axis early
+    if axis not in (0, -1):
+        raise NotImplementedError(
+            "OpenVINO logspace supports axis=0 or axis=-1 only"
+        )
```

Comment:
While technically this is correct, this is now inconsistent with how `linspace` is handled.